packages/firestore/test/util/api_helpers.ts

```diff
@@ -55,11 +60,19 @@ export function firestore(): Firestore {
 }
 
 export function collectionReference(path: string): CollectionReference {
-  return new CollectionReference(pathFrom(path), firestore());
+  return new CollectionReference(
+    pathFrom(path),
+    firestore(),
+    defaultDocumentDataConverter()
```

Comment:
We made converter optional, so perhaps you can drop all of these (and get rid of `defaultDocumentDataConverter` entirely)?